easybuild/easyconfigs/o/OR-Tools/OR-Tools-7.1-fix-makefiles.patch

```diff
@@ -0,0 +1,81 @@
+# Use EB deps and install prefix
+# wpoely86@gmail.com
+diff -Nur or-tools-7.1.orig/Makefile.local.eb or-tools-7.1/Makefile.local.eb
+--- or-tools-7.1.orig/Makefile.local.eb	1970-01-01 01:00:00.000000000 +0100
++++ or-tools-7.1/Makefile.local.eb	2019-05-20 20:49:11.941360942 +0200
+@@ -0,0 +1,5 @@
++UNIX_SWIG_BINARY = $(EBROOTSWIG)/bin/swig
++JAVA_HOME = $(EBROOTJAVA)
++UNIX_PYTHON_VER = 3.7
++UNIX_GUROBI_DIR=$(EBROOTGUROBI)
++GUROBI_LIB_VERSION=$(shell echo $(EBVERSIONGUROBI) | sed 's:\([0-9]\+\)\.\([0-9]\+\)\..*:\1\2:')
```

Comment:
Sort these alphabetically (easier for humans to digest)?

But is adding this `Makefile.local.eb` and making the `Makefile` include it actually needed though?

Can't you just define these via `buildopts`, like this:

```python
buildopts = "UNIX_SWIG_BINARY=${EBROOTSWIG}/bin/swig JAVA_HOME=${EBROOTJAVA} ..."
```

Reply:
they will not overwrite the contents of `Makefile.local`

Reply:
Hmm, that surprises me to be honest. Did you figure out exactly why that doesn't work?

Reply:
It's changed.